ts/src/base/Exchange.ts

```diff
@@ -2404,6 +2405,21 @@ export default class Exchange {
         throw new NotSupported (this.id + ' setMargin() is not supported yet');
     }
 
+    async fetchMarginAdjustmentHistory (symbol: Str = undefined, type: Str = undefined, since: Num = undefined, limit: Num = undefined, params = {}): Promise<object[]> {
```

Comment:
@samgermain this return type will not work properly in c#, can you add a proper type for this method?